lib/windowsbasetest.pm

```diff
@@ -237,4 +246,15 @@ sub power_configuration {
     );
 }
 
+sub password_expired {
+    # There's need to type the old password and the new one twice and then press
+    # enter an additional time for acknowledging.
+    send_key 'ret';    # Ok message for starting the process
+    my @i = (0 .. 2);
+    for (@i) {
```

Comment:
Nitpicking here but apply only if we don't need to restart VR.

```suggestion
    foreach my $i (0 .. 2) {
```

Reply:
Actually you don't use the iterator so you can just use

```suggestion
    for (0 .. 2) {
```

Reply:
Thanks for your suggestions :)